tests/python/test_transformer_engine.py

```diff
@@ -68,28 +81,58 @@ def test_transformer_layer(multidevice_test, benchmark, compute_type):
     match compute_type:
         case ComputeType.FORWARD:
 
-            def benchmark_fn():
-                return transformer_layer(x)
+            def benchmark_fn(profile):
+                if profile:
+                    torch.cuda.cudart().cudaProfilerStart()
+
+                y = transformer_layer(x)
+                torch.cuda.synchronize()
```

Comment:
I totally forgot about having to synchronize. The report timing makes much more sense now. Phew!